### DISCUSSION TITLE
Initial pytest 'no module test' error

### DISCUSSION THREAD

**Discussion by zaxhutchinson** [lava-nc/lava#252] [Q&A]
First install. Followed linux instructions. Ran into an error(?) running pytest. Got the following error and coverage failed at 39%.

```
_________________________________________________ ERROR collecting tests/lava/tutorials/test_tutorials.py __________________________________________________
ImportError while importing test module '/home/zax/Projects/lava/tests/lava/tutorials/test_tutorials.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/lava/tutorials/test_tutorials.py:13: in <module>
    from test import support
E   ModuleNotFoundError: No module named 'test'
```

I don't know much about the test module. I can't even import it on my system.

I changed the call in __name__ == "__main__" to just:
```unittest.main()```
since the class inherits from unittest. And it worked (89% coverage). But I'm not sure if the original call had different magic.

OS: Manjaro
Python: 3.10.4

**Comment by awintel** [comment]
Best to file errors like these as issues instead of opening a discussion. Thanks.

**Comment by mgkwill** [comment]
I doubt this is a bug, but rather an issue with the environment in Manjaro. @zaxhutchinson Try installing python-tests and try again. 

In some cases it looks like Arch based distros don't install the correct python-tests dependency. See the old comment in this thread https://aur.archlinux.org/packages/python-iniparse "Without the package python-tests I have an error when I try to install this package. Maybe this should be added to the dependency list?"
